frontend/src/component/admin/instance-privacy/index.tsx

```diff
@@ -0,0 +1,151 @@
+        },
+    };
+
+export const InstancePrivacyAdmin = () => {
+    const { settings } = useTelemetry();
+    const { uiConfig, loading } = useUiConfig();
+
+    if (loading) {
+        return null;
+    }
+
+    const versionActivenessInfo = settings?.versionInfoCollectionEnabled
+        ? versionCollectionActivenessManagementTexts.enabled
+        : versionCollectionActivenessManagementTexts.disabled;
+
+    const featureActivenessInfo = settings?.featureInfoCollectionEnabled
+        ? featureCollectionActivenessManagementTexts.enabled
+        : featureCollectionActivenessManagementTexts.disabled;
+
+    let dependsOnFeatureCollection: undefined | String = undefined;
+    if (!settings?.versionInfoCollectionEnabled)
+        dependsOnFeatureCollection = settings?.featureInfoCollectionEnabled
+            ? 'Note: Feature Usage Collection is enabled, but for it to be active you must also enable Version Info Collection'
+            : 'When you enable Feature Usage Collection you must also enable Version Info Collection';
+
+    return (
+        <PageContent header={<PageHeader title="Instance Privacy" />}>
+            <StyledBox>
+                <InstancePrivacy
+                    title={versionCollectionDetails.title}
+                    infoText={versionCollectionDetails.infoText}
+                    concreteDetails={versionCollectionDetails.concreteDetails}
+                    enabled={settings?.versionInfoCollectionEnabled}
+                    changeInfoText={versionActivenessInfo.changeInfoText}
+                    variablesTexts={versionActivenessInfo.environmentVariables}
+                />
+                <ConditionallyRender
+                    condition={Boolean(
+                        uiConfig.flags.experimentalExtendedTelemetry
+                    )}
+                    show={
+                        <InstancePrivacy
+                            title={featureCollectionDetails.title}
+                            infoText={featureCollectionDetails.infoText}
+                            concreteDetails={
+                                featureCollectionDetails.concreteDetails
+                            }
+                            enabled={
+                                settings?.featureInfoCollectionEnabled &&
+                                settings?.versionInfoCollectionEnabled
+                            }
+                            changeInfoText={
+                                featureActivenessInfo.changeInfoText
+                            }
+                            variablesTexts={
+                                featureActivenessInfo.environmentVariables
+                            }
+                            dependsOnText={dependsOnFeatureCollection}
+                        />
+                    }
+                />
+            </StyledBox>
+        </PageContent>
+    );
+};
```

Comment:
We don't usually use `index` as a filename, especially not for components. I think maybe what you meant is that this one should be called `InstancePrivacy` and maybe the existing `InstancePrivacy` should be renamed to `InstancePrivacySection` or something similar.